docs/source/concept_models/vivarium_swissre_cervicalcancer/cervical_cancer_concept_model.rst

```diff
@@ -0,0 +1,597 @@
+.. _cervical_cancer_concept_model:
+
+=======================================
+Vivarium CSU Cervical Cancer Simulation
+=======================================
+
+.. contents::
+  :local:
+
+.. list-table:: List of abbreviations
+   :header-rows: 1
+
+   * - Label
+     - Definition
+   * - BCC
+     - benign cervical cancer
+   * - CC
+     - cervical cancer 
+   * - ACMR
+     - all-cause mortality rate
+   * - prev_c432
+     - prevalence of cervical cancer
+   * - incidence_c432
+     - incidence of cervical cancer
+   * - csmr_c432
+     - cause-specific mortality rate of cervical cancer
+
+.. _1.0:
+
+1.0 Background
+++++++++++++++
+
+.. _1.1:
+
+1.1 Project overview
+--------------------
+This project will generate forecasts of cervical cancer mortality and morbidity 
+to allow Swiss Re to identify trends that are important to its business 
+decision-making. IHME will produce both a baseline (business as usual) forecast, 
+and an alternative scenario forecast in which key cervical cancer screening 
+practice and HPV vaccination are implemented in a simulation framework. Baseline 
+forecasts will incorporate expected trends in relevant risk factors. Alternative 
+scenario forecasts will incorporate baseline forecasts and the expected impact of 
+new screening technologies and vaccination. All forecasts will represent the Swiss 
+Re’s insured population from the weighted blend of Chinese provinces. Forecasts 
+run from year 2020 to 2040.
+
+.. _1.2:
+
+1.2 Literature review
+---------------------
+
+.. todo::
+
+ add more literature background
+
+
+.. _2.0:
+
+2.0 Modeling aims and objectives
+++++++++++++++++++++++++++++++++
+IHME will estimate the yearly number of cases of benign and invasive cervical 
+cancer detected under specific cervical cancer screening practices and the yearly 
+number of deaths from undetected cervical cancer (both in unit of per 100,000 
+insured person-years) in order to identify pay-out trends for cervical cancer 
+claims.
+
+
+.. _3.0:
+
+3.0 Causal framework
+++++++++++++++++++++
+
+.. _3.1:
+
+3.1 Causal variables
+--------------------
+ 
+Outcome
+ - Cervical cancer detection (benign and invasive)
+ - Mortality and morbidity of cervical cancer
+Most proximal determinant/exposure
+ - Cervical cancer status
+ - Cervical cancer screening
+ - HPV infection  
+Confounders
+ - age
+ - sex
+Effect modifiers
+ - N/A
+Mediators
+ - N/A
+
+
+.. _4.0:
+
+4.0 Intervention
+++++++++++++++++
+Scale-up of both cervical cancer screening coverage and HPV vaccination rate 
+among insured population
+
+.. _4.1:
+
+4.1 Simulation scenarios
+------------------------
+**Baseline:** project existing level of HPV vaccination for women aged 15 to 44 
+and cervical cancer screening for women aged 25 to 64, where:
+ - Women without HPV infection are provided with cytology every three years
+ - Women with HPV infection are provided cytology every year
+
+**Alternative scenario:** by 2040, linear ramp up both HPV vaccination to cover 
```

Comment:
To get the extremes, I think we might need a scenario with constant screening and scaled-up vaccination and a scenario with scaled-up screening and constant vaccination

Reply:
That sounds like a lowest claim and highest claim scenario, scale-up both interventions are recommended by WHO initiative on 2030 cervical cancer elimination strategy since ramp up screening or vaccination alone is not as efficacious as combination interventions.